File: backend/music/src/main/kotlin/at/htl/entity/Artist.java
package at.htl.entity;

import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity;

import javax.persistence.Entity;

@Entity
public class Artist extends PanacheEntity {
    public String firstName;

    public String lastName;

    public String alias;

    public String info;

    public Artist() {
    }

    public Artist(String firstName, String lastName, String alias, String info) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.alias = alias;
        this.info = info;
    }
}

File: backend/music/src/main/kotlin/at/htl/entity/Artist.kt
package at.htl.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
class Artist() {
    @Id
    @GeneratedValue
    var id: Long? = null;
    lateinit var firstName: String;
    lateinit var lastName: String;
    lateinit var alias: String;
    lateinit var info: String;

    constructor(firstName: String, lastName: String, alias: String, info: String) : this() {
        this.firstName = firstName
        this.lastName = lastName
        this.alias = alias
        this.info = info
    }
}